client/components/data/domain-management/index.jsx

```diff
@@ -105,20 +61,17 @@ class DomainManagementData extends React.Component {
 				{ needsProductsList && <QueryProductsList /> }
 
 				<CalypsoShoppingCartProvider>
-					<StoreConnection
-						component={ this.props.component }
-						context={ this.props.context }
-						currentUser={ this.props.currentUser }
-						domains={ this.props.domains }
-						getStateFromStores={ getStateFromStores }
-						hasSiteDomainsLoaded={ this.props.hasSiteDomainsLoaded }
-						isRequestingSiteDomains={ this.props.isRequestingSiteDomains }
-						products={ this.props.productsList }
-						selectedDomainName={ this.props.selectedDomainName }
-						selectedSite={ selectedSite }
-						sitePlans={ this.props.sitePlans }
-						stores={ stores }
-					/>
+					{ React.createElement( this.props.component, {
+						context: this.props.context,
+						domains: this.props.selectedSite ? this.props.domains : null,
+						hasSiteDomainsLoaded: this.props.hasSiteDomainsLoaded,
+						isRequestingSiteDomains: this.props.isRequestingSiteDomains,
+						products: this.props.products,
+						selectedDomainName: this.props.selectedDomainName,
+						selectedSite: this.props.selectedSite,
```

Comment:
nit: `selectedSite` is already destructured a few lines above.

Reply:
> nit: `selectedSite` is already destructured a few lines above.

Addressed in #51856.